apps/web/vibes/soul/primitives/compare-card/index.tsx

```diff
@@ -1,15 +1,15 @@
 import { clsx } from 'clsx';
 
+import { ButtonLink } from '@/vibes/soul/primitives/button-link';
 import {
   type Product,
   ProductCard,
   ProductCardSkeleton,
 } from '@/vibes/soul/primitives/product-card';
 import { Rating } from '@/vibes/soul/primitives/rating';
+import { Reveal } from '@/vibes/soul/primitives/reveal';
```

Comment:
I think this dependency needs to be added to the primitive component declaration